core-bundle/src/Resources/contao/library/Contao/Email.php

```diff
@@ -355,13 +402,27 @@ public function sendTo()
 			return false;
 		}
 
-		$this->objMessage->setTo($arrRecipients);
-		$this->objMessage->setCharset($this->strCharset);
+		if ($this->objMessage instanceof EmailMessage)
+		{
+			$this->objMessage->to(...$arrRecipients);
+		}
+		else
+		{
+			$this->objMessage->setTo($arrRecipients);
+			$this->objMessage->setCharset($this->strCharset);
```

Comment:
Why is it necessary to set the charset here?

Reply:
That's the original code of the `Email` class. The Email class allows you to set the Charset yourself - and thus it needs to be passed to the message object as well. For the Symfony Message this is done later one when actually setting the text or html content.